### PR TITLE
[Merged by Bors] - chore: split Data.Real.Basic

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: o.pdf
Comment (hrmacbeth) [inline o.pdf:1]: Is this file meant to be included?

Comment (semorrison) [reply o.pdf:1]: Oops, thanks. Leftovers from `lake exe graph`.

File: Mathlib/Data/Real/Basic.lean [hunk 1]
```diff
@@ -4,7 +4,6 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Floris van Doorn
 -/
 import Mathlib.Algebra.Bounds
-import Mathlib.Algebra.Order.Archimedean
 import Mathlib.Algebra.Star.Basic
 import Mathlib.Data.Real.CauSeqCompletion
```
Comment (sgouezel) [inline Mathlib/Data/Real/Basic.lean:9]: Could you mention in the file-level docstring what is done in this file, and where one may find the conditionally complete linear order instance? (This is one of the most basic things on reals, so users may be surprised not to find it here).

**Review by sgouezel — COMMENTED** [review]
bors d+
Thanks!

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**